src/libertem/udf/base.py

```diff
@@ -1601,6 +1672,16 @@ def from_udfs(
         kwargs = [udf._kwargs for udf in udfs]
         return cls(kwargs, roi, corrections, tiling_scheme)
 
+    def update_from_udfs(self, udfs):
+        self._kwargs = [udf._kwargs for udf in udfs]
+
+    def update_parameters(self, parameters):
+        self._kwargs = parameters
```

Comment:
See above, not covered and apparently only used in one place?

Reply:
Removed.